buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -77,230 +56,120 @@ limitations under the License."""
 open class IdePlugin : Plugin<Project> {
 
     override fun apply(project: Project): Unit = project.run {
-        configureIdeaForRootProject()
+        if (System.getProperty("idea.active", "false")!!.toBoolean()) {
```

Comment:
:x: should use `providers.systemProperty("idea.active")` in order to let instant execution consider that property as a build logic input